bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| externalContact                  | unitConfig      | External contact                                                 | Number                   |      | 1 -6 (function 1-6)                                                                                  |
+| errorOutputFunction              | unitConfig      | Error output function (collective error or just error)           | Number                   |      | 1 = collective error, 2 = only error                                                                 |
+| filterChange                     | unitConfig      | Filter change                                                    | Switch                   |      |                                                                                                      |
+| filterChangeInterval             | unitConfig      | Filter change interval in months                                 | Number                   |      | 1 - 12                                                                                               |
+| filterChangeRemainingTime        | general         | Filter change remaining time in minutes                          | Number:Time              | min  | 1 - 55000                                                                                            |
+| filterChangeReset                | general         | Reset filter change remaining time                               | Switch                   |      | ON = reset                                                                                           |
+| bypassRoomTemperature            | unitConfig      | Bypass room temperature in °C                                    | Number:Temperature       | °C   | 10 - 40                                                                                              |
+| bypassMinOutsideTemperature      | unitConfig      | Bypass outside temperature in °C                                 | Number:Temperature       | °C   | 5 - 20                                                                                               |
+| tbd                              | general         | No description available for this parameter in the specification | Number:Temperature       | °C   | 3 - 10                                                                                               |
+| factorySettingWzu                | general         | Factory setting WZU                                              | Switch                   |      | ON = reset individual switching times                                                                |
+| factoryReset                     | general         | Factory reset                                                    | Switch                   |      | ON = reset start                                                                                     |

```

Comment:
Actually OH is not intended to configure Things, including factory resets. Is there a use case for these parameters?

Reply:
Personally, I didn't use them. I merely added this since it was part of the protocol specification.
Probably it will not do any harm to remove it.